app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -155,6 +155,19 @@ def alert_field_changed
       end
     end
 
+    if params[:value_mw_greater_than]
```

Comment:
These params come from the automated expressions defined at:
https://github.com/lucasponce/manageiq/blob/408a08defb5a8ae0e2c89eb9cff0240aeed1ea38/app/models/miq_alert.rb#L366
I tried to follow similar style to define the names of the parameters included.
Also naming the specific parameters used for middleware automated expressions to not collide with other automated expressions.